docs/source/_templates/autoapi/python/class.rst

```diff
@@ -44,7 +44,7 @@ This is currently not possible, see https://github.com/readthedocs/sphinx-autoap
    {% if visible_overview_methods %}
 
    .. admonition:: Methods
-      :class: admoition-methods
+      :class: admonition-methods
```

Comment:
Is this ok?

Reply:
There currently is no CSS rule for the class `admonition-methods` (neither in our custom CSS file nor in the CSS files from the theme), so we could also remove it without changing the appearance of the boxes.

Reply:
Ok, thanks. Then I'll remove it.